framework/src/Volo.Abp.Swashbuckle/wwwroot/swagger/ui/abp.swagger.js

```diff
@@ -28,7 +28,16 @@ var abp = abp || {};
             }
             // Intercept .well-known request when the discoveryEndpoint is provided
             if (!firstRequest && oidcDiscoveryEndpoint.length !== 0 && request.url.includes(".well-known/openid-configuration")) {
-                request.url = oidcDiscoveryEndpoint;
+                if (oidcDiscoveryEndpoint.endsWith(".well-known/openid-configuration")) {
+                    request.url = oidcDiscoveryEndpoint;
+                    console.log(request.url);
```

Comment:
Should we use `abp.log.debug` instead of `console.log`?

Reply:
why not just remove it

Reply:
https://github.com/abpframework/abp/pull/17434/files#diff-383bbd9f9228466840c24c0c2c619e2ba1eb008ad0554a3e23a55fc1880b6a69R190

Reply:
could any one explain why can't remove the `console.log`?

Reply:
> could any one explain why can't remove the `console.log`?

So the users can see the URL and understand if they get an error because of malformed URL:
![image](https://github.com/abpframework/abp/assets/25867/1c671fe2-4153-47d9-a939-39362f2d38bd)